lib/core/constant/strings.dart

```diff
@@ -33,9 +33,19 @@ class AppStrings{
 
 
   // images path
+  static const burgerIcon= 'assets/search_screen_image/burger.png';
+  static const tacoIcon= 'assets/search_screen_image/taco.png';
+  static const drinkIcon= 'assets/search_screen_image/drink.png';
+  static const pizzaIcon= 'assets/search_screen_image/pizza.png';
 
   static const googleIcon = 'assets/AuthImages/google.svg';
   static const facebookIcon = 'assets/AuthImages/facebook.svg';
   static const appleIcon = 'assets/AuthImages/apple.svg';
```

Comment:
**suggestion**: create a new class called AssetImages and keep this in that file